### ISSUE TITLE
make usage

### ISSUE THREAD

**Issue by gedw99** [Darkness4/auth-htmx#11]
I like your make file approach...

For me it's not automagically installing and  go tools though... I think I am missing something :)

For example the cow tool... 

```
make watch
gow -e=go,mod,html,tmpl,env,local run ./main.go
make: gow: No such file or directory
make: *** [Makefile:36: watch] Error 127
```

using gnu make on Darwin btw:
``` 
which make
/opt/homebrew/opt/make/libexec/gnubin/make
```

**Comment by gedw99** [comment]
Ended up with:

``` 
cd tools && go mod download
cd tools && cat tools.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
```

**Comment by Darkness4** [comment]
Oops, yeah, I forgot to call the variable in the Makefie. Need to sleep tho. I will fix this in the next 24h.

**Comment by Darkness4** [comment]
Fixed with 0ccc6389dd9f700dc9b18c228aa71159378e9b4a.
